src/components/trade/Price.tsx

```diff
@@ -272,9 +278,9 @@ export const Price: React.FC<Props> = ({
           <SwapPrice
             baseToken={baseToken}
             quoteToken={quoteToken}
-            isPriceInverted
-            showBaseToken
+            isPriceInverted={!wasPriorityAdjusted}
```

Comment:
This make it very difficult to understand, why `isPriceInverted = !wasPriorityAdjusted` 🤯